htdocs/product/reassort.php

```diff
@@ -123,7 +123,7 @@
 if (! empty($conf->global->PRODUCT_USE_UNITS)) $sql.= ', u.short_label as unit_short';
 $sql.= ' FROM '.MAIN_DB_PREFIX.'product as p';
 $sql.= ' LEFT JOIN '.MAIN_DB_PREFIX.'product_stock as s on p.rowid = s.fk_product';
-$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX.'entrepot as e on s.fk_entrepot = e.rowid AND e.entity IN ('.getEntity('entrepot').')';
+$sql .= ' INNER JOIN '.MAIN_DB_PREFIX.'entrepot as e on s.fk_entrepot = e.rowid AND e.entity IN ('.getEntity('entrepot').')';
```

Comment:
If you introduce a INNER JOIN here instead of LEFT, it means the if there is a record into llx_product and not into product_stock (because there is no stock at all), no line will be returned for the product.
But we expect to have it. If a product has a expected value of 10 and quantity of 0, we must have it.